cypress/plugins/index.js

```diff
@@ -16,7 +16,19 @@
  * @type {Cypress.PluginConfig}
  */
 // eslint-disable-next-line no-unused-vars
-module.exports = (on, config) => {
-  // `on` is used to hook into various events Cypress emits
-  // `config` is the resolved Cypress config
-}
+  module.exports = (on, config) => {
+    // `on` is used to hook into various events Cypress emits
+    // `config` is the resolved Cypress config
+    on('task', {
+      log(message) {
+        console.log(message)
+  
+        return null
+      },
+      table(message) {
+        console.table(message)
+  
+        return null
+      }
```

Comment:
is this being used? usually we try to get rid of console messages.

Reply:
From what I understand, it is so that you can see the same error messages in console output from the Cypress testing UI.

https://github.com/component-driven/cypress-axe#standard-output

Reply:
is there a need to return null ?

Reply:
is there a need to return null ?

Reply:
It appears to be the typical implementation for the task event listener in Cypress

https://docs.cypress.io/api/commands/task#Usage